staking-pool/Cargo.toml

```
@@ -1,26 +1,37 @@
 [package]
-name = "staking-contract"
+name = "staking-pool"
 version = "0.1.0"
-authors = ["Illia Polosukhin <ilblackdragon@gmail.com>"]
+authors = ["Near Inc <hello@nearprotocol.com>"]
```

```suggestion
authors = ["Near Inc <hello@near.org>"]
```
Does this apply here too?